### PR TITLE
[RFR] Add more doc for newcomers

### PR DESCRIPTION
**Pull request by fzaninotto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # marmelab-boilerplate
 
+A starter kit for new apps, based on:
+
+* ES6 everywhere (with some bits of ES7, e.g. spread operator on objects)
+* React.js, React Router, and Redux (for the frontend)
+* Angular.js and ng-afmin (for the admin)
+* Node.js and Koa.js and PostgreSQL (for the API server)
```
Comment (Kmaschta) [inline README.md]: Node.js, Koa.js and PostgreSQL

File: README.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+* Makefile, webpack and Mocha (for the build)
```
Comment (Kmaschta) [inline README.md]: Build and test

File: README.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+* CORS support (including on IE8, thanks to `xDomain`)
+* Fully automated start and stop (see `Makefile`)
+* Frontend app built with React, `redux`, `redux-saga`, `react-router`
+* Using `react-dev-tools` for easier debugging in development
+* SASS preprocessor (using `node-sass`)
+* Including a non-trivial example with several routes, Ajax calls, and functional tests
+* Fully automated build with `webpack`, including development (`webpack-dev-server`) and production target
+* Admin app built with Angular and `ng-admin`
+* Including a full-featured admin panel with references
+* Unified test build, running unit and functional tests on all apps, powered by `mocha`, `selenium`
```
Comment (Kmaschta) [inline README.md]: And nightwatch ?

File: src/api/lib/rateLimiter.js [hunk 4]
```diff
@@ -1,5 +1,8 @@
 /**
  * Simple in-memory rate limiter
+ *
+ * WARNING: DO NOT USE IN PRODUCTION, IT EXPOSES THE APP TO DDOS
```
Comment (Kmaschta) [inline src/api/lib/rateLimiter.js]: :+1:

File: README.md (hunk 3 above)
Comment (fzaninotto) [inline README.md]: I didn't see we used nightwatch. Why do we use both nightwatch and selenium?

File: README.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
 # marmelab-boilerplate
 
+A starter kit for new apps, based on:
+
+* ES6 everywhere (with some bits of ES7, e.g. spread operator on objects)
+* React.js, React Router, and Redux (for the frontend)
+* Angular.js and ng-afmin (for the admin)
```
Comment (Kmaschta) [inline README.md]: s/ng-adfmin/ng-admin/

File: README.md (hunk 3 above)
Comment (djhi) [inline README.md]: We wanted to use nightwatch to pilot selenium for frontend functional tests

Comment (Kmaschta) [inline README.md]: No idea, @djhi and @alexisjanvier have set up the functional tests, but I think @djhi skipped it.

Comment (fzaninotto) [inline README.md]: I confirm, I don't see it used. I'll open a ticket.

**Comment by fzaninotto** [comment]
Code review applied

**Pull request merged**